File: Services/Admin/Admin.Domain/Review_Banner.cs
namespace Admin.Domain;

public class Review_Banner
{
    public int review_id { get; set; }
    public int media_id { get; set; }
    public static Review_Banner Create(int review_id,int media_id)
    {
        return new Review_Banner
        {
            review_id = review_id,
            media_id = media_id
        };
    }
}

File: Services/DB_Handler/models/ReviewBanner.py
from sqlalchemy import Column, Integer, ForeignKey
from sqlalchemy.orm import relationship
from .base import Base


class ReviewBanner(Base):
    __tablename__ = 'review_banner'

    review_id = Column(Integer, ForeignKey("review.review_id"), primary_key=True)
    media_id = Column(Integer, primary_key=True)  # No FK to media

    review = relationship("Review", back_populates="banners")
